iati/tests/test_validate.py

```diff
@@ -0,0 +1,215 @@
+"""A module containing tests for data validation."""
+import pytest
+import iati.core.data
+import iati.core.default
+import iati.core.schemas
+import iati.core.tests.utilities
+import iati.validate
+
+
+class TestValidate(object):
```

Comment:
If all tests are marked xfail currently, would be better to mark class as xfail instead.
https://docs.pytest.org/en/latest/skipping.html#skip-all-test-functions-of-a-class-or-module (also relevant to xfail)